lms/models/upload.py

```diff
@@ -51,10 +55,17 @@ def _run_auto_checks(solution: Solution) -> None:
 
 
 def new(user: User, file: FileStorage) -> Tuple[List[int], List[int]]:
+    solution_hash = hashing.by_file(file)
```

Comment:
You can delete this line.
You can see that the new extractor provides the solution hash for each exercise.